tox.ini

```
@@ -13,7 +12,8 @@ passenv = MYSQL_*
 setenv =
   COVERAGE_FILE=.coverage.{envname}
 deps = mock
-       pytest
+       py27: pytest < 5.0
+       py{36,37}: pytest
```

How does this work with py38 ? Maybe just as a dependency to the other packages? To be explicit, I would suggest the following. 
```suggestion
       !py27: pytest
```